regions/shapes/point.py

```diff
@@ -105,15 +105,22 @@ def as_patch(self, origin=(0, 0), **kwargs):
         patch : `~matplotlib.patches.Circle`
             Matplotlib circle patch
         """
-        # FIXME: need to make radius constant
-        from matplotlib.patches import Circle
+        from matplotlib import pyplot as plt
+        from matplotlib.lines import Line2D
 
-        mpl_params = self.mpl_properties_default('patch')
+        if ax is None:
+            ax = plt.gca()
+
+        # We can move this to a method like `as_artist`
+        mpl_params = self.mpl_properties_default('Line2D')
         mpl_params.update(kwargs)
+        point = Line2D([self.center.x + origin[0]],
+                       [self.center.y + origin[1]],
+                       **mpl_params)
 
-        return Circle((self.center.x, self.center.y), radius=2, **mpl_params)
+        return point
 
-    def plot(self, origin=(0, 0), ax=None, **kwargs):
+    def plot(self, ax=None, origin=(0, 0), **kwargs):
```

Comment:
This should be removed.

Reply:
indeed, it's now redundant with the core plot method